src/wild-category-theory/isomorphisms-in-noncoherent-large-wild-infinity-infinity-precategories.lagda.md

```diff
@@ -0,0 +1,227 @@
+# Isomorphisms in noncoherent large wild (∞,∞)-categories
+
+```agda
+{-# OPTIONS --guardedness #-}
+
+module wild-category-theory.isomorphisms-in-noncoherent-large-wild-infinity-infinity-precategories where
+```
+
+<details><summary>Imports</summary>
+
+```agda
+open import foundation.dependent-pair-types
+open import foundation.universe-levels
+
+open import wild-category-theory.isomorphisms-in-noncoherent-wild-infinity-infinity-precategories
+open import wild-category-theory.noncoherent-large-wild-infinity-infinity-precategories
+```
+
+</details>
+
+## Idea
+
+Consider a
+[noncoherent large wild $(∞,∞)$-precategory](wild-category-theory.noncoherent-large-wild-infinity-infinity-precategories.md)
+𝒞. An
+{{#concept "isomorphism" Disambiguation="in noncoherent large wild (∞,∞)-precategories" Agda=is-iso-Noncoherent-Large-Wild-⟨∞,∞⟩-Precategory}}
+in 𝒞 is a morphism `f : x → y` in 𝒞 [equipped](foundation.structure.md) with
+
+- a morphism `s : y → x`
+- a $2$-morphism `is-split-epi : f ∘ s → id`, where `∘` and `id` denote
+  composition of morphisms and the identity morphism given by the transitive and
+  reflexive structure on the underlying
+  [globular type](structured-types.globular-types.md), respectively
+- a proof `is-iso-is-split-epi : is-iso is-split-epi`, which shows that the
+  above $2$-morphism is itself an isomorphism
+- a morphism `r : y → x`
+- a $2$-morphism `is-split-mono : r ∘ f → id`
+- a proof `is-iso-is-split-mono : is-iso is-split-mono`.
+
+This definition of an isomorphism mirrors the definition of
+[biinvertable maps](foundation-core.equivalences.md) between types.
+
+It would be in the spirit of the library to first define what split epimorphisms
+and split monomorphisms are, and then define isomorphisms as those morphisms
+which are both. When attempting that definition, one runs into the problem that
+the $2$-morphisms in the definitions should still be isomorphisms.
```

Comment:
We could have written a mutually coinductive definition I think, but I don't believe Agda allows that to span multiple files